power app sample/readme.md

```diff
@@ -0,0 +1,76 @@
+![banner](https://i.ibb.co/QQCVq4m/banner.png)
```

Comment:
I've opted for using https://imgbb.com/ for direct image hosting because there is no restriction on how long the photos can stay hosted there and also I wanted to keep the repository folder itself as minimal as possible. We can switch to hosting the images on the repo itself if that is the preferred style though. Let me know.

Reply:
Sounds good